src/docs/goose-book/src/config/cookie.md

```diff
@@ -0,0 +1,10 @@
+# Cookies
+
+By default, Goose enables the Reqwest `cookies` feature. However, if you are running a load test that doesn't use cookies, you can disable this feature. This disables the feature in Reqwest, and allows an optimization during Goose startup (see https://github.com/tag1consulting/goose/pull/557 for more information).
+ 
+To disable client cookies and optimize startup performance, disable default features and pick a tls client in `Cargo.toml`, for example:
+
+```toml
+[dependencies]
+goose = { version = "^0.17", default-features = false, features = ["reqwest/default-tls"] }
```

Comment:
When copying the example for testing the branch, I got this error below. Perhaps there should be additional cargo features that only enable `default-tls`/`rustls-tls` without including `cookies`?

```
error: failed to parse manifest at `/home/michael/devenv/src/work/futo/matrix-goose/Cargo.toml`

Caused by:
  feature `reqwest/default-tls` in dependency `goose` is not allowed to contain slashes
  If you want to enable features of a transitive dependency, the direct dependency needs to re-export those features from the `[features]` table.
```